jenkins/custom-ci/generic/motr.groovy

```diff
@@ -118,11 +131,15 @@ pipeline {
                             fi
                         fi
                         export build_number=${CUSTOM_CI_BUILD_ID}
-                        make rpms
+                        if [ "${os_version}" == "rockylinux-8.4" ]; then
+                            make rpms
+                        else
+                            make deb
+                        fi            
                     '''
             }
         }
-        
+
         stage ('Copy RPMS') {
```

Comment:
We need to disable all subsequent stages like `Copy RPMs` etc. for `ubuntu` as of now. It will break once Ubuntu packaging starts working. 
Maybe we can skip custom-ci support for now.

Reply:
@shailesh-vaidya, added conditional checks on stages for OS

Reply:
Can you please disable CC build stage also at https://github.com/gauravchaudhari02/cortx-re/blob/motr-build-ubuntu/jenkins/custom-ci/generic/motr.groovy#L191 
Also, I feel we should add CC build in parallel execution.

Reply:
@shailesh-vaidya, why to disable cortx-cc? 
CC is already in parallel execution

Reply:
Sorry missed it. No changes needed for CC